packages/react-client/src/hooks/public.ts

```diff
@@ -3,11 +3,7 @@ export { useReconnection } from "./useReconnection";
 export { useCamera, useMicrophone, useInitializeDevices } from "./useDevices";
 export { useParticipants } from "./useParticipants";
 export { useScreenShare } from "./useScreenShare";
-export { useStatus } from "./useState";
+export { useStatus } from "./useStatus";
+export { useINTERNAL_FishjamClient } from "./useFishjamClient";
```

Comment:
Idea: It's possible to export this hook from:
```ts
import { useINTERNAL_FishjamClient } from '@fishjam-cloud/react-client/internal';  
```
instead of

```ts
import { useINTERNAL_FishjamClient } from '@fishjam-cloud/react-client';  
```

WDYT? Is it worth investing additional time to make this change?

Reply:
If you know how to make it work that way - then go for it.

Reply:
done